src/robot_interface/robot_interface.py

```diff
@@ -136,3 +136,19 @@ def get_telemetry_publishers(self, queue: Queue, robot_id: str) -> List[Thread]:
 
         """
         raise NotImplementedError
+
+    @abstractmethod
+    def is_robot_online(self) -> bool:
```

Comment:
Would it be better to have an indication of robot status? That way we can get a response indicating Offline or not, but also incorporate Available and Busy. It may be that the robot is online, but unavalable

Reply:
Yes, implemented an enum which reflects the status from Flotilla.